clang/docs/ReleaseNotes.rst

```diff
@@ -438,6 +438,36 @@ Moved checkers
 Sanitizers
 ----------
 
+- Added the ``-fsanitize-undefined-ignore-overflow-pattern`` flag which can be
+  used to disable specific overflow-dependent code patterns. The supported
+  patterns are: ``add-overflow-test``, ``negated-unsigned-const``, and
+  ``post-decr-while``. The sanitizer instrumentation can be toggled off for all
```

Comment:
it does not state what is the default state of the flag.

Reply:
It is not explicitly stated but no patterns are excluded by default. This is somewhat evident by "... _can be_ used to disable" and "sanitizer instrumentation _can be_ toggled off".

Reply:
I understand. I am actually asking to add explicit statement. E.g. `The dafault is none`.
or none (default)

Reply:
Ok. I will open a PR to expand the documentation following your review.